zk/stages/stage_sequence_execute.go

```diff
@@ -339,21 +340,26 @@ func SpawnSequencingStage(
 		batchState.onBuiltBlock(blockNumber)
 
 		// commit block data here so it is accessible in other threads
-		if errCommitAndStart := sdb.CommitAndStart(); errCommitAndStart != nil {
-			return errCommitAndStart
+		if !batchState.isL1Recovery() {
```

Comment:
I think this could break the Async verification, because block's data will not be visible in other threads.

Reply:
That's fine in an L1 recovery scenario.  We only ran with executors for recovery as a sense check on the witness going to the remote executor.

Reply:
yeah, if we do that (testing witness again), we might as well just comment-out this `if`